ksql-test-runner fails loading hamcrest and other testing jars

When executing the `ksql-test-runner` from a packaged confluent installation, the following error is thrown:

$ ./ksql-test-runner -s /tmp/statements.sql -i /tmp/input.json -o /tmp/output.json
```
...
Caused by: java.lang.ClassNotFoundException: org.hamcrest.Matcher
    at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```

The reason is because the following jars are not packaged in KSQL:
```
ksql-functional-tests/hamcrest-all-1.3.jar
ksql-functional-tests/junit-4.12.jar
```

In 5.3.1, the `kafka-clients-5.3.1-css` is not found either:
```
ksql-functional-tests/kafka-clients-5.3.2-ccs-20190919.011153-4-test.jar
```
**Workaround**
The workaround is to copy the required jars into the confluent/ksql classpath. For instance, being `/opt/confluent/confluent-5.3.1/share/java/ksql` the classpath:

```
$ cd /opt/confluent/confluent-5.3.1/share/java/ksql
$ wget https://repo1.maven.org/maven2/junit/junit/4.12/junit-4.12.jar
$ wget https://repo1.maven.org/maven2/org/hamcrest/hamcrest-all/1.3/hamcrest-all-1.3.jar
```

This bug is found since the `5.3.x` branch.

--- a/ksql-package/pom.xml
+++ b/ksql-package/pom.xml
@@ -80,6 +80,33 @@
             <version>${project.version}</version>
         </dependency>
 
+        <!-- Required for ksql-test-runner -->
+
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-all</artifactId>
+            <scope>compile</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>compile</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <classifier>test</classifier>
+            <scope>compile</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <scope>compile</scope>
+        </dependency>
+
       <!-- Required for running tests -->
 
       <dependency>

--- a/ksql-package/src/assembly/package.xml
+++ b/ksql-package/src/assembly/package.xml
@@ -83,7 +83,6 @@
             <excludes>
                 <exclude>io.confluent:rest-utils</exclude>
                 <exclude>io.confluent:common-config</exclude>
-                <exclude>io.confluent:common-utils</exclude>
                 <exclude>io.confluent:common-metrics</exclude>
             </excludes>
         </dependencySet>

